src/browserbehavior/DefaultBrowserBehavior.ts

```diff
@@ -70,6 +70,16 @@ export default class DefaultBrowserBehavior implements BrowserBehavior {
     return this.isFirefox();
   }
 
+  supportsCanvasCapturedStreamPlayback(): boolean {
+    return !this.isIOSSafari();
+  }
+
+  supportsVideoProcessingBasedOnTensorflowModel(): boolean {
+    // most of the tf models have memory leak on Firefox
+    // on android chrome, the performance is not "real-time conferencing"
+    return !this.isAndroid() && this.isChrome();
```

Comment:
You almost certainly mean `this.chromeLike.includes(this.browser.name)` instead of `this.isChrome()`, otherwise you exclude Electron, Edge, ….

Reply:
this is removed along with segmentation filter demo